Standalone/selenium.conf

```diff
@@ -61,7 +61,7 @@ stderr_capture_maxbytes=50MB
 
 [program:selenium-standalone]
 priority=15
-command=bash -c "/opt/bin/start-selenium-standalone.sh && kill -s SIGINT `cat /var/run/supervisor/supervisord.pid`"
+command=bash -c "/opt/bin/start-selenium-standalone.sh; EXIT_CODE=$?; kill -s SIGINT `cat /var/run/supervisor/supervisord.pid`; exit $EXIT_CODE"
```

Comment:
Using `exit $EXIT_CODE` to make the exit status of `start-selenium-standalone.sh` visible in logs, otherwise it would be the exit status of the kill command.
```
seluser@faf20ce83332:/etc/supervisor/conf.d$ /opt/bin/entry_point.sh
...
2022-10-24 10:40:06,323 INFO spawned: 'selenium-node' with pid 1144
2022-10-24 10:40:06,330 INFO success: selenium-node entered RUNNING state, process has stayed up for > than 0 seconds (startsecs)
2022-10-24 10:40:06,339 INFO exited: selenium-node (exit status 1; not expected)
2022-10-24 10:40:06,339 WARN received SIGINT indicating exit request
...
```